spec/Interoperation.tex

```diff
@@ -118,14 +128,24 @@ \subsection{Calling Chapel Functions}
     function-body
 
 external-name:
-  string-literal
+  expression
 \end{syntax}
 
-If the optional \sntx{external-name} is
-supplied, then it is used verbatim as the exported function symbol.  Otherwise,
-the Chapel name of the procedure is exported.  The rest of the procedure
-declaration is the same as for a non-exported function.  An exported procedure can be
-called from within Chapel as well.  Currently, iterators cannot be exported.
+The rest of the procedure declaration is the same as for a non-exported
+function.  An exported procedure can be called from within Chapel as
+well. Currently, iterators cannot be exported.
+
+As with the \snxt{extern-name} for \chpl{extern} \chpl{proc}, if this
+syntax element is provided, then it is a \chpl{param} \chpl{string}
```

Comment:
"then it is a param string determining the name" -> "then it must be a param string and will be used to determine the name"